cub/benchmarks/bench/select/unique.cu

```diff
@@ -57,14 +55,15 @@ struct policy_hub_t
 template <typename T, typename OffsetT, typename MayAlias>
 static void unique(nvbench::state& state, nvbench::type_list<T, OffsetT, MayAlias>)
 {
-  using input_it_t         = const T*;
-  using flag_it_t          = cub::NullType*;
-  using output_it_t        = T*;
-  using num_selected_it_t  = OffsetT*;
-  using select_op_t        = cub::NullType;
-  using equality_op_t      = ::cuda::std::equal_to<>;
-  using offset_t           = OffsetT;
-  constexpr bool may_alias = MayAlias::value;
+  using input_it_t        = const T*;
+  using flag_it_t         = cub::NullType*;
+  using output_it_t       = T*;
+  using num_selected_it_t = OffsetT*;
+  using select_op_t       = cub::NullType;
+  using equality_op_t     = ::cuda::std::equal_to<>;
+  using offset_t          = OffsetT;
+  constexpr cub::SelectionOption selection_option =
+    MayAlias::value ? cub::SelectionOption::SelectPotentiallyInPlace : cub::SelectionOption::Select;
```

Comment:
Question: This seems to be a common selection choice, should that be wrapped in a function?